packages/integration-tests/scripts/bridgeERC20.ts

```diff
@@ -174,7 +175,7 @@ const TOKEN_AMOUNT = '10';
   console.log('Get message proof on Fuel...');
   const withdrawMessageProof = await fuelAcct.provider.getMessageProof(
     fWithdrawTxResult.id,
-    messageOutReceipt.messageId,
+    messageOutReceipt.nonce,
```

Comment:
Has there been a renaming?
Seems like a `messageId` and `nonce` may be 2 different things.

Reply:
yes they are different. change was intentional

Reply:
as sdk now expects nonce